apis/pkg/v1alpha1/provider_types.go

```diff
@@ -45,6 +45,11 @@ type Provider struct {
 // Crossplane.
 type ProviderSpec struct {
 	PackageSpec `json:",inline"`
+
+	// ControllerConfigRef references a ControllerConfig resource that will be
+	// used to configure the packaged controller Deployment.
+	// +optional
+	ControllerConfigRef *v1alpha1.Reference `json:"controllerConfigRef,omitempty"`
```

Comment:
I think the prevalent pattern is for reference fields to be named `FooReference` despite using `fooRef` as their JSON tag.